Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -1238,10 +1238,19 @@ def elabToAdditive : Syntax → CoreM Config
         as there is only one declaration for the attributes.\n\
         Instead, you can write the attributes in the usual way."
     trace[to_additive_detail] "attributes: {attrs}; reorder arguments: {reorder}"
+    let doc ← doc.mapM fun
+      | `(str|$doc:str) => do
+        -- TODO: deprecate `str` docstring syntax in Mathlib
+        return doc.getString
```

Comment:
This is not the same behavior as before, strictly speaking (as this returns `""` rather than `none` in a corner case I can't make happen)